docs/NNNN-mutable-head.md

```diff
@@ -0,0 +1,439 @@
+# OCFL Community Extension NNNN: Mutable HEAD
+
+  * **Extension Name:** NNNN-mutable-head
+  * **Authors:** Peter Winckles
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+In the core OCFL specification, every mutation of an object is recorded as a new
+immutable version. This extension enables an OCFL client to create a HEAD
+version of an object that is able to be mutated in-place without generating
+additional versions. This allows changes to an object to be accumulated over a
+period of time within an object's OCFL object root before they are committed to
+an immutable version, reducing version inflation and providing a client-agnostic
+mechanism for interpreting content that has not yet been officially versioned in
+OCFL.
+
+## Important Note
+
+Changes should be left in the mutable HEAD for as short of time as possible
+before committing them into an immutable OCFL version. OCFL clients that do not
+implement this extension CANNOT interact with content in the mutable HEAD, and
+the creation of new versions before the mutable HEAD is committed invalidates
+the contents of the mutable HEAD.
+
+## Specification
+
+### Terminology
+
+* **Mutable HEAD:** A mutable OCFL version that contains the HEAD state of an
+  object and is outside of the core OCFL specification.
+* **Mutable HEAD version:** The version identifier of the mutable HEAD, eg `v3`
+* **Root HEAD version:** The version identifier of the most recent immutable
+  OCFL version under the object root.
+* **Extension directory:** This extension’s root directory within objects that
+  use it, `[object-root]/extensions/NNNN-mutable-head`.
+* **Mutable HEAD version directory:** The directory within the extension
+  directory that contains the OCFL version for the mutable HEAD,
+  `[object-root]/extensions/NNNN-mutable-head/head`.
+* **Mutable HEAD inventory:** The inventory located in the mutable HEAD version
+  directory.
+* **Root inventory:** The inventory located in the OCFL object’s root.
+* **Mutable HEAD content directory:** The version content directory within the
+  mutable HEAD version directory,
+  `[object-root]/extensions/NNNN-mutable-head/head/content`.
+* **Commit:** Move the mutable HEAD out of the extension directory and into the
```

Comment:
Minor: For linguistic consistency across terms:
> [The action of moving] the mutable HEAD...

Reply:
I don't quite follow how you'd like me to change the wording

Reply:
I think Andrew is suggesting changing "Move" to "The action of moving" because that is a more consistent way to describe what "Commit" means